modules/nw-sriov-example-vf-function-in-pod.adoc

```diff
@@ -9,7 +9,7 @@ You can run a remote direct memory access (RDMA) or a Data Plane Development Kit
 
```

Comment:
s/= Example use of a virtual function in a Pod/= Example use of a virtual function in a pod/
s/application in a Pod with SR-IOV VF/application in a pod with SR-IOV VF/